README.md

```diff
@@ -1,5 +1,53 @@
-# Heroku Cloud Native PHP Buildpack
+# Heroku Cloud Native Buildpack: PHP
 
-[![CI](https://github.com/heroku/buildpacks-php/actions/workflows/ci.yml/badge.svg)](https://github.com/heroku/buildpacks-php/actions/workflows/ci.yml)
+[![Cloud Native Buildpacks Registry: heroku/php][registry-badge]][registry-url]
```

Comment:
should we also advertise the buildpack api version here?

Reply:
I don't think it's useful for most consumers, plus we'd have to keep it up to date here in the README.md. The buildpack registry doesn't even display that (though, maybe it should). I think anyone that knows enough about buildpacks to know there is a Buildpack API version probably knows to look for it in buildpack.toml.

Reply:
> The buildpack registry doesn't even display that (though, maybe it should)

xref https://github.com/buildpacks/registry-api/issues/124